.github/workflows/ci.yml

```diff
@@ -74,9 +74,15 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install --requirement requirements.txt
       - name: Build docs
-        run: python -m mkdocs build --clean --site-dir artifacts/docs --config-file mkdocs.yml --strict
+        run: ./build.ps1 docs
+        shell: pwsh
       - name: Upload docs
         uses: actions/upload-artifact@v4
         with:
           name: documentation
           path: artifacts/docs
+      - name: Upload link check report
+        uses: actions/upload-artifact@v4
+        with:
+          name: docs-link-check
+          path: artifacts/docs-link-check.html

```

Comment:
Can we not zip it? It would be more convenient, and it's really small anyway

Reply:
Apparently it's not an option :(
There's a `compression-level` you can set to 0, but if I understand correctly it still produces an archive...

Reply:
I'd've preferred not to archive as well. Most of the time we won't need to look at the file anyhow (we'll have good links and/or could check the build output) and downloading and clicking right into the archive in Windows File Explorer worked for me.